src/openapi/streaming/connection/transport/signalr-transport.js

```diff
@@ -37,6 +37,9 @@ function mapConnectionState(state) {
  */
 function handleError(errorDetail) {
     log.error(LOG_AREA, 'Transport error', errorDetail);
+    if (errorDetail && errorDetail.source && errorDetail.source.status === 401) {

```

Comment:
I looked at the signal-r source and also our logs to work out this condition 🤞

Reply:
I only see 401. this one depends on what open api returns, which doesn't seem to ever be 403.
really the token refresh should probably be 401 only too.